Fixing presidential net values

noahmanger opened the pull request:
Resolves https://github.com/18F/openFEC-web-app/issues/750

Ping @LindsayYoung . This look right?

jmcarp commented on static/templates/filings/candidate.hbs:
```diff
@@ -56,14 +56,28 @@
+      <td class="panel__data">{{{currency net_contributions_cycle_to_date}}}</td>
+    </tr>
+    {{/if}}
+    {{#if net_operation_expenditures_period}}
```
"net_operating_expenditures_period"?

LindsayYoung replied:
Sorry I did not write up that ticket with enough specificity. The change I was talking about was specific to presidential, and not all candidates and it is for the reports rather than totals. 

The new variable is called "net_operating_expenditures_cycle_to_date" and the name of it should be similar. 

It should go in the reports slide out of the filings view:
![screen shot 2015-11-17 at 9 39 00 am](https://cloud.githubusercontent.com/assets/4406333/11214409/139ae1a6-8d0f-11e5-9b32-29c80100d3e0.png)

I will make a PR to this branch.

LindsayYoung replied:
So we used to show all financial summary information for reports on the website, when we were showing most recent report. I remember doing a through check to make sure we had everything. It looks like we don't show that information anymore on the website. 

I guess we can close this. I do wish all the data was accessible somewhere on the website, perhaps we could do a reports view in the future?

noahmanger replied:
@jmcarp thanks for the catch. Fixed it.

@LindsayYoung this template is for the reports slide out on the filings view (the screenshot you posted). This is the template that's used for candidate reports. I wrapped it in a conditional so that it will only show if the report doesn't have the `net_contributions_period`

noahmanger replied:
As for your second point, i don't understand what's different. Isn't this report view showing all financial summary information from the report? If it's not, this is where it should be.

noahmanger commented:
Made my comments on a line comment that got removed when I changed it. So here they are again:

@jmcarp thanks for the catch. Fixed it.

@LindsayYoung this template is for the reports slide out on the filings view (the screenshot you posted). This is the template that's used for candidate reports. I wrapped it in a conditional so that it will only show if the report doesn't have the `net_contributions_period` / `net_operating_expenditures_period`.

As for your second point, i don't understand what's different. Isn't this report view showing all financial summary information from the report? If it's not, this is where it should be.

jmcarp commented:
What's the status of this patch @noahmanger @LindsayYoung?

LindsayYoung commented:
@noahmanger this is missing a lot of summary information- there are 3 different kinds of summary information from the respective forms of the Presidential, House/Senate and PACs/Parties. 

For example on a presidential candidate:

Here is what we have:
![screen shot 2015-11-19 at 11 41 28 am](https://cloud.githubusercontent.com/assets/4406333/11277029/9f587006-8eb2-11e5-8959-99313f227509.png)

Here are all of the fields:
![screen shot 2015-11-19 at 11 42 19 am](https://cloud.githubusercontent.com/assets/4406333/11277032/a37cd276-8eb2-11e5-9459-d9630e12f478.png)

noahmanger commented:
Gotcha. Thanks for clarifying. Does this patch at least solve the initial problem you identified?

I think we should open up a new open issue for fleshing out the rest of the fields.

noahmanger commented:
Discussed this with @LindsayYoung and it should be good to go now.
